spark/pom.xml

```diff
@@ -198,6 +198,9 @@
         <!-- profile spark-x only affect the embedded spark version in zeppelin distribution -->
         <profile>
             <id>spark-2.4</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
             <properties>
                 <spark.version>2.4.3</spark.version>
```

Comment:
maybe it makes sense to bump spark dependency to 2.4.4 as well?

Reply:
Make sense, I will do it